addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -29,16 +29,17 @@ if (_vehicle isEqualType objNull) then {_vehicle = typeOf _vehicle};
 if (_vehicle == "") exitWith {
     ERROR_1("VehicleType [%1] is empty in ace_rearm_fnc_addVehicleMagazinesToSupply",_string);
 };
+private _turrets = [_vehicle] call FUNC(getAllRearmTurrets);
```

Comment:
Yeah, it's the function call itself that causes the performance overhead.

```
private _a = allTurrets veh;
_a pushBack [-1];
_a
```
is as fast as
`[[-1]] + allTurrets veh`

But since it's better to have a function anyway (cleaner code, used in many places, intended as replacement for `allTurrets`), it doesn't really matter what is inside it.